# -*- coding: utf-8 -*-
# Generated by Django 1.10.5 on 2017-04-17 10:39
from __future__ import unicode_literals

import datetime
import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('match', '0007_auto_20170403_0038'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='robotics2',
            name='total_time',
        ),
        migrations.AddField(
            model_name='robotics2',
            name='used_time',
            field=models.DurationField(default=datetime.timedelta(0, 150), validators=[django.core.validators.MaxValueValidator(datetime.timedelta(0, 150))], verbose_name='temps utilisé'),
        ),
        migrations.AlterField(
            model_name='robotics1',
            name='team',
            field=models.OneToOneField(error_messages={'unique': 'Résultat déjà saisi pour cette équipe.'}, on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='teams.Team', verbose_name='équipe'),
        ),
        migrations.AlterField(
            model_name='robotics2',
            name='team',
            field=models.OneToOneField(error_messages={'unique': 'Résultat déjà saisi pour cette équipe.'}, on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='teams.Team', verbose_name='équipe'),
        ),
        migrations.AlterField(
            model_name='robotics3',
            name='team',
            field=models.OneToOneField(error_messages={'unique': 'Résultat déjà saisi pour cette équipe.'}, on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='teams.Team', verbose_name='équipe'),
        ),
    ]
